src/Service/SourcesConverter.php

```diff
@@ -72,7 +72,7 @@ public function convertSourcesJsonToSourceSelectionItems(string $sources): array
                 ]
             );
 
-            $sourceSelectionItems[$item['sku']] = $sourceSelectionItem;
+            $sourceSelectionItems[] = $sourceSelectionItem;
```

Comment:
This change makes sense, does anything else consume `convertSourcesJsonToSourceSelectionItems` that may need its handling updated? 🤔